content/datasets/_index.md

```diff
@@ -9,5 +9,11 @@ The Open Force Field Initiative makes a number of resources available to the com
 
 ---
 
-- [open-forcefield-data](https://github.com/openforcefield/open-forcefield-data): Curated datasets from the NIST ThermoML database to be used for force field parameterization.
+- [open-forcefield-data](https://github.com/openforcefield/open-forcefield-data): 
+  * Curated datasets from the NIST ThermoML database to be used for force field parameterization:
+    + 
```

Comment:
There's an extra `+` here on a line by itself.

Reply:
Fixed. I copied the wrong file version and missed it.